app/components/bookings/create/create-booking.component.html

```diff
@@ -0,0 +1,39 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ _workpackage.stringId }} - {{ _workpackage.name }}</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+    <button accent floating block full *ngIf="!_bookingSession" (click)="createBookingSession()" >
+      <ion-icon name="ios-stopwatch-outline"></ion-icon>Livebuchung starten
+    </button>
+    <button accent floating block full *ngIf="_bookingSession" (click)="checkoutBookingSession()" >
```

Comment:
The point is that if a booking session is running and you want to make another booking for another day for example, you can do both.
Otherwise you start the session, and you will only be able to book and end this session.